source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACOHostConfigPrivate.h

```diff
@@ -11,6 +11,7 @@ using namespace AdaptiveCards;
 
 @interface ACOHostConfig()
 
+- (instancetype)init:(std::shared_ptr<HostConfig> const &)config;
```

Comment:
>init: [](start = 16, length = 5)

Also minor: initializers that take arguments are usually named initWithSomething. In this case, `initWithConfig`.

Reply:
thank you

---
In reply to: [169506801](https://github.com/Microsoft/AdaptiveCards/pull/1143#discussion_r169506801) [](ancestors = 169506801)